sql/003_create_nonprofits_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `non_profits` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  `city` varchar(45) NOT NULL,
+  `zipcode` int NOT NULL,
+  `points` int NOT NULL,
+  `total_volunteers` int NOT NULL,
+  `total_hours` int NOT NULL,
+  `approved_hours` int NOT NULL,
```

Comment:
are we going to record approved and pending hours as a running total or per any given day?

Reply:
kenthia says it is a running total